tracker-radar-tests/TR-domain-matching/tracker_radar_reference.json

```diff
@@ -31,6 +31,44 @@
                 }
             ]
         },
+        "sometimes-bad.third-party.site": {
+            "domain": "sometimes-bad.third-party.site",
+            "owner": {
+                "name": "Test Site for Tracker Blocking",
+                "displayName": "Bad Third Party Site",
+                "privacyPolicy": "",
+                "url": "http://bad.third-party.site"
+            },
+            "prevalence": 0.1,
+            "fingerprinting": 3,
+            "cookies": 0.1,
+            "categories": [],
+            "default": "ignore",
```

Comment:
I think we should also test options in the `default:block` scenario because it slightly flips the logic, right?

Reply:
We shouldn't have that case. I'll add a test for that, probably in the actual list building code.